generic3g/specs/BracketSpec.F90

```diff
@@ -47,7 +47,9 @@ module mapl3g_BracketSpec
 
       procedure :: make_adapters
       procedure :: set_geometry
+! #ifndef __GFORTRAN__
```

Comment:
```suggestion
```
I think these should all be eliminated at this point.   Just keep the one for generic ...

Reply:
Done.